src/graph/graph_hub.c

```diff
@@ -326,8 +323,70 @@ int UpdateNodeLabels
 
 	// Update label matrixes.
 	Graph_LabelNode(gc->g, node->id ,label_ids, label_count);
+	
+	// cleanup
+	array_free(label_ids);
```

Comment:
is this a leak we didn't noticed?

Reply:
this is GitHub showing new changes from some reason